.github/ISSUE_TEMPLATE/docDB.yml

```diff
@@ -0,0 +1,14 @@
+name: Core-DocDB
+description: Choose this template to create a DocDB issue.
+title: "[docDB] Title"
```

Comment:
nit: DocDB

Reply:
@bmatican Thank you! Made the necessary change.